### PR TITLE
Reorganize API Gateway deployments in containers Complements #2538

### PR DESCRIPTION
**Pull request by andreamussap**
Reorganize API Gateway deployments in containers. Complements #2538

http://pr-2578.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_installation/apigw_containers/deployment_flows/

### CODE REVIEW

File: content/en/docs/apim_installation/apigw_containers/deployment_flows/axway_image_deployment/helm_deployment.md [hunk 1]
```diff
@@ -457,6 +457,10 @@ aga:
   enabled: false
 ```
 
+{{< alert title="Note" color="primary" >}}
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apigw_containers/deployment_flows/axway_image_deployment/helm_deployment.md:460]: @vobrien-axway @pfoleyAx do you want to keep the **sample** both as text in the page and as a option to download?

http://pr-2578.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_installation/apigw_containers/deployment_flows/axway_image_deployment/helm_deployment/#sample-customized-values-file

File: content/en/docs/apim_installation/apigw_containers/container_migration.md [hunk 2]
```diff
@@ -91,10 +91,10 @@ Alternatively, to create a new keyspace for the API Gateway container, follow th
...
 
 {{< alert title="Note" color="primary" >}}You can also export your API Manager APIs as an API collection if you require a backup.{{< /alert >}}
 
 ### Step 5 – Deploy helm chart
 
-In order to deploy your custom configurations, the helm chart must first be deployed using the default configurations. Once the containers come up, your custom configurations can be copied into the containers before restarting them. See [Install API Gateway using your customized YAML file](/docs/apim_installation/apigw_containers/deployment_flows/axway_image_deployment/helm_deployment#install-api-gateway-using-your-customized-yaml-file)
\ No newline at end of file
+To deploy your custom configurations, the Helm chart must first be deployed using the default configurations. After the containers come up, your custom configurations can be copied into the containers before restarting them. For more information, see [Install API Gateway using your customized YAML file](/docs/apim_installation/apigw_containers/deployment_flows/axway_image_deployment/helm_deployment#install-api-gateway-using-your-customized-yaml-file).
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apigw_containers/container_migration.md:100]: (After the containers come up) --> what does 'come up' mean, start !?

File: content/en/docs/apim_installation/apigw_containers/deployment_flows/axway_image_deployment/helm_deployment.md (hunk 1 above)
Comment (pfoleyAx) [reply content/en/docs/apim_installation/apigw_containers/deployment_flows/axway_image_deployment/helm_deployment.md:460]: Just an option to download

File: content/en/docs/apim_installation/apigw_containers/container_migration.md (hunk 2 above)
Comment (pfoleyAx) [reply content/en/docs/apim_installation/apigw_containers/container_migration.md:100]: "come up" is equivalent to "start"

File: content/en/docs/apim_installation/apigw_containers/deployment_flows/axway_image_deployment/helm_deployment.md (hunk 1 above)
Comment (andreamussap) [reply content/en/docs/apim_installation/apigw_containers/deployment_flows/axway_image_deployment/helm_deployment.md:460]: Done! Deleted the hard coded example and left just the option to download the code sample

**Pull request merged**